en/docs/learn/fraud-detection.md

```diff
@@ -54,12 +62,29 @@ Many enterprises use identity verification (IDV) services to ensure that users p
   systems to retrieve accurate information. Apart from using trusted sources, these services also engage artificial
   intelligence and machine learning techniques to gather behavioral insights about the users.
 
+### Business Intelligence
+
+Business intelligence (BI) comprises the strategies and technologies used by enterprises for the data analysis of
+business information. It helps to investigate insights and trends about the business for scaling and adapting the
+system. BI systems analyze business data and provide information in a presentable formats which helps relevant
+authorities to make better business decisions. These BI systems make use of advanced technologies such as data
+warehouses, dashboards, ad hoc reporting, data discovery tools and cloud data services. They consume sales,
+production, financial and many other source of business data in order to provide predictive insights to the business.
+ Further they collect information corresponding to other enterprises in the same industry for benchmarking.
+
+Some use-cases where BI systems are applicable are listed below.
+
+   -   Aggregating user data such as there attributes (location, device, gender etc), preferences and behavioral
```

Comment:
their attributes including location, device, gender, and so on, preferences, and behavioral patterns to provide product recommendations.